add_pseudocolor_function

HaleySchuhl opened the pull request:
Added the pseudocolor function to the PSII tutorial and updated the index. Shouldn't get merged into master until we merge the pseudocolor function into PlantCV. Right now it creates a FvFm grayscale image but when the pcv.fluor_fvfm function also returns the grayscale image (rather than just printing it out) we can get rid of the calculations to provide FvFm image.

HaleySchuhl committed:
add_pseudocolor_function

Added the pseudocolor function to the PSII tutorial and updated the index. Shouldn't get merged into master until we marge the pseudocolor function into PlantCV.

nfahlgren commented:
@HaleySchuhl, this builds the notebook environment with the PyPI build of PlantCV right? I'm thinking we have to make a new release so that the `pseudocolor` function will be included in PyPI.

HaleySchuhl commented:
@nfahlgren Yes, I'm pretty sure it does. We could always wait on this pull request until I update the pcv.analyze functions so the PyPI build includes the `pcv.pseudocolor` function.

HaleySchuhl committed:
analyze_function_updates

- Updated the tutorials that use analyze_objects and those using fluor_fvfm functions to reflect removing the filename parameter, and how to access the array of images returned by each function.

HaleySchuhl committed:
update analyze_bound* functions

Changed the VIS and VIS/NIR tutorials to reflect the changes to analyze_bound_horizontal and analyze_bound_vertical since they no longer take a filename parameter

HaleySchuhl committed:
update tutorials with analyze_nir_intensity function

Revised tutorials containing the analyze_nir_intensity function since it no longer takes a filename parameter.

HaleySchuhl committed:
update analyze_color useage

Updated the VIS and VIS/NIR tutorials to reflect the changes made to the analyze_color function

nfahlgren reviewed (approved):
v3.1.0 is up on PyPI now, so we're good to go!
